README.md

```diff
@@ -1,24 +1,23 @@
 # Home
 
-Alibaba ACK Distro (ACK distribution) is the Kubernetes distribution released by Alibaba for heterogeneous IaaS environment. Users can get complete content and long-term support for free through Alicloud ACR. Its components have been verified and security checked by Alicloud ACK service and Alibaba Group's core business scenarios in large-scale production environment, with industry-leading security and reliability.
-​
+Alibaba ACK Distro is a Kubernetes distribution based on Alibaba Container Service for Kubernetes to support production ready Kubernetes deployment for heterogeneous IaaS environment. Users can get complete content and long-term support for free through Alibaba Cloud. Its components have been verified and security checked by Alibaba Cloud Container Service for Kubernetes (ACK) and Alibaba Group's core business scenarios in large-scale production environment, with industry-leading security and reliability.
```

Comment:
"Alibaba ACK Distro is a Kubernetes distribution based on Alibaba Container Service for Kubernetes to support production ready Kubernetes deployment for heterogeneous IaaS environment." This sentence is kind of weird, I suggest refining to "Alibaba ACK Distro is a Kubernetes distribution based on Alibaba ACK service to support  Kubernetes deployment for heterogeneous IaaS environment.